src/app/lib/views/movie_detail.js

```diff
@@ -47,7 +47,7 @@
 
       curSynopsis = {old: '', crew: '', cast: '', allcast: '', vstatus: null};
 
-      if (!this.model.get('synopsis') || !this.model.get('rating') || this.model.get('rating') == '0' || this.model.get('rating') == '0.0' || !this.model.get('runtime') || this.model.get('runtime') == '0' || !this.model.get('trailer') || !this.model.get('cover') || this.model.get('cover') == 'images/posterholder.png' || !this.model.get('backdrop') || this.model.get('backdrop') == 'images/posterholder.png' || (Settings.translateSynopsis && Settings.language != 'en')) {
+      if (!this.model.get('synopsis') || !this.model.get('rating') || this.model.get('rating') == '0' || this.model.get('rating') == '0.0' || !this.model.get('runtime') || this.model.get('runtime') == '0' || !this.model.get('trailer') || !this.model.get('poster') || this.model.get('poster') == 'images/posterholder.png' || !this.model.get('backdrop') || this.model.get('backdrop') == 'images/posterholder.png' || (Settings.translateSynopsis && Settings.language != 'en')) {
```

Comment:
could you add a comment explaining why we are checking these seemingly arbitrary values?

Reply:
Ahh, this is mine :D Checking for missing metadata and fetching them? (could also be added in the movie_list etc but will create a lot of calls to TMDb.

Reply:
That makes sense to me, can you add a comment in the code explaining basically that? This way nobody needs to come searching for the PR comments to find out why we're doing this :)